pandas/core/indexes/multi.py

```diff
@@ -811,7 +811,9 @@ def duplicated(self, keep='first'):
 
     @Appender(ibase._index_shared_docs['fillna'])
     def fillna(self, value=None, downcast=None):
-        # isna is not implemented for MultiIndex
+        """
+        isna is not implemented for MultiIndex
```

Comment:
isna -> fillna

Reply:
@jorisvandenbossche please  see PR #18269 for changes related  to `fillna`.